crates/aide/src/axum/mod.rs

```diff
@@ -338,6 +364,38 @@ where
         self
     }
 
+    #[cfg(feature = "axum-extra")]
+    /// Create a route to the given method router with trailing slash removal and include it in
+    /// the API documentation.
+    /// 
+    /// This method accepts a transform function to edit
+    /// the generated API documentation with.
+    /// 
+    /// See [`axum_extra::routing::RouterExt::route_with_tsr`] for details.
+    #[tracing::instrument(skip_all, fields(%path))]
+    pub fn api_route_with_tsr_with(
```

Comment:
The double `_with` seems a bit awkward, maybe we can replace the second one with `and`? Would read a bit more naturally to me, but really not sure.

Reply:
Maybe just api_route_with_tsr

Reply:
`api_route_with_tsr` is already in use.
I agree with @jplatte that the double `with` isn't great and I think the `and` name is a pretty good name, but I'm also unsure if this follows best practices around naming functions. To me the `and` sounds more like the `Option::and()` or `Result::and` method.

Reply:
> api_route_with_tsr is already in use.
At first glance, I misread `tsr_with` was from RouteExt but it's not. I see what you mean now. Then `_and` will be good names.


> To me the and sounds more like the Option::and() or Result::and method
I don't think it will confuse with Option::and or Result::and